stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -116,7 +116,7 @@ isposdef(x::Number) = imag(x)==0 && real(x) > 0
 # a tuple containing 1 and a cumulative product of the first N-1 sizes
 # this definition is also used for StridedReshapedArray and StridedReinterpretedArray
 # which have the same memory storage as Array
-function stride(a::Union{DenseArray,StridedReshapedArray,StridedReinterpretArray}, i::Int)
+function stride(a::Union{Array,StridedReshapedArray,StridedReinterpretArray}, i::Int)
```

Comment:
Why this change?

Reply:
Ah, because `DenseArray` didn't use to mean contiguous in memory... but now it does https://github.com/JuliaLang/julia/pull/26085

Reply:
Bump